azavea/commonhealth-data-map
# Implement region selection

## Overview

This integrates region selection changes from the EAC repository.

This includes:
- Implement region selection on click and country masking.
- Keep the mask for the capacity tab at EAC, regardless of country selection
- Add zoom functionality to the region selection - selecting a region from the dropdown or clicking will zoom to that country.
- Show a pointer over clickable countries.
- Don't show a popup on the mobility tab over countries that have no mobility data.

__Note:__ This PR depends on #54, so that will need to be merged first; I'll integrate any changes made to that PR as part of review.

### Checklist


```console
$ chan added "A feature with no bugs. [#00](https://github.com/azavea/commonhealth-data-map/pull/00)"
```

## Testing Instructions

On the EAC dashboard:
- click on EAC countries and see that the regions are selected, with the appropriate mask being laid on the map and the map zooming to the selected country.
- Capacity tab does not have region selection, so no clicking is enabled and the EAC mask always shows.

On the TCP dashboard
- No clicking is enabled as the only region in the dropdown is 'Global'.

Also note that the popups for the mobility tab only show for countries that have data.

Closes #28
